gatsby-browser.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import {GatsbyBrowser} from 'gatsby';
 
+import {FeedbackWidget} from 'sentry-docs/components/feedback/feedbackWidget';
 import PageContext from 'sentry-docs/components/pageContext';
 
 export const wrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
   element,
   props: {pageContext},
-}) => <PageContext.Provider value={pageContext}>{element}</PageContext.Provider>;
+}) => <PageContext.Provider value={pageContext}>
+      <React.Fragment>
```

Comment:
nit: shouldn't need a fragment here because it's inside the context provider

do we want an ErrorBoundary though? maybe here or inside the component.